spec/models/importers/organisation_spec.rb

```diff
@@ -1,19 +1,40 @@
 require 'rails_helper'
```

Comment:
that is fine.. will update.. I thought it clear enough.. cheers for point that out